src/textual/screen.py

```diff
@@ -976,17 +976,6 @@ def _forward_event(self, event: events.Event) -> None:
                 else:
                     widget._forward_event(event._apply_offset(-region.x, -region.y))
 
-        elif isinstance(event, (events.MouseScrollDown, events.MouseScrollUp)):
```

Comment:
`MouseScrollUp` and `MouseScrollDown` inherit from `MouseEvent` so this branch is never taken.
(See the branch immediately before.)